sources/DropZoneTab.cpp

```diff
@@ -46,7 +51,7 @@ DropZone::_Init()
 
 DropZone::DropZone(bool replicatable)
 	:
-	BView("Filer dropzone", B_WILL_DRAW | B_FULL_UPDATE_ON_RESIZE),
+	BView(B_TRANSLATE("Filer dropzone"), B_WILL_DRAW | B_FULL_UPDATE_ON_RESIZE),
 	fReplicated(false)
```

Comment:
This is just the internal name of the BView and shouldn't be translated. It not only creates unneeded work for the translators - which can be dismissed, really, but if someone wants to script the GUI with 'hey' these internal names mustn't change with the locale.